cmd/root.go

```diff
@@ -1,47 +1,33 @@
 package cmd
 
 import (
+	"fmt"
 	"os"
 
 	"github.com/spf13/cobra"
 )
 
+var (
+	cfgFile string
+	reAuth  bool
+)
 
-
-// rootCmd represents the base command when called without any subcommands
-var rootCmd = &cobra.Command{
+var RootCmd = &cobra.Command{
 	Use:   "fleet-scheduler",
```

Comment:
can we rename this project to just `scheduler`, since it could scheduler more than just fleet things like firmware syncs for example